lib/appenders/index.js

```diff
@@ -6,6 +6,11 @@ const levels = require('../levels');
 const layouts = require('../layouts');
 const adapters = require('./adapters');
 
+const isNode =
+  typeof process !== 'undefined' &&
+  process.versions != null &&
+  process.versions.node != null;
+
```

Comment:
Just a nitpick, but why check against `null`?
I would expect:

```js
const isNode = typeof process !== 'undefined' && process.versions && process.versions.node
```
```

Reply:
Good call. I should have seen this.